sklearn/decomposition/dict_learning.py

```diff
@@ -713,8 +714,7 @@ def dict_learning_online(X, n_components=2, alpha=1, n_iter=100,
         .. versionadded:: 0.20
 
     method_max_iter : int, optional (default=1000)
-        It is passed to the underlying ``method`` as their ``max_iter``
-        parameter.
+        Maximum number of iterations to perform in each ``sparse_encode`` step.
```

Comment:
To be consistent with `dict_learning`:

```
Maximum number of iterations to perform.
```

Reply:
The reason for the extra part at the end is that `dict_learning_online` has an `n_iter` parameter as well, which has `Maximum number of iterations to perform.` as the description. I needed to distinguish the two somehow.

Reply:
With two iters, we may need to describe what they actually do. Since `dict_learning_online` only supports `lassos`, maybe:

```
n_iter:
    Number of mini-batch iterations to perform.

method_max_iter:
    Maximum number of iterations to perform when solving the lasso problem.
```